pkgs/development/python-modules/yubico/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, pytest, pyusb }:
+
+buildPythonPackage rec {
+  pname   = "python-yubico";
+  version = "1.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gd3an1cdcq328nr1c9ijrsf32v0crv6dgq7knld8m9cadj517c7";
+  };
+
+  checkInputs = [ pytest ];
```

```suggestion
  checkInputs = [ pytestCheckHook ];
```
Please convert the checkPhase